tests/integration/runner

```diff
@@ -156,6 +156,8 @@ if __name__ == "__main__":
                 env_tags += "-e {}={} ".format("DOCKER_POSTGRESQL_JAVA_CLIENT_TAG", tag)
             elif image == "yandex/clickhouse-integration-test":
                 env_tags += "-e {}={}".format("DOCKER_BASE_TAG", tag)
```

Comment:
```suggestion
                env_tags += "-e {}={} ".format("DOCKER_BASE_TAG", tag)
```

Reply:
@tavplubix  many thanks and sorry for the disturbance.